modules/deleting-service-cli.adoc

```diff
@@ -6,13 +6,13 @@
 [id="deleting-service-cli_{context}"]
 = Deleting a service from the CLI
 
-You can delete the add-on services from your {product-title} cluster through the OCM CLI.
+You can delete the add-on services from your {product-title} cluster through the OpenShift Cluster Manager CLI.
```

Comment:
The ocm CLI isn't supported yet - this should be deleted from the doc.

Reply:
I can provide you background, I had to do the same in the OpenShift Cluster Manager doc.